docs/modules/releasenotes/partials/release-notes-26.1.adoc

```diff
@@ -42,3 +42,23 @@ These methods are:
 * `cssClassAsArray`: splits a `cssClass`-string into an array of css classes
 
 In addition, similar methods have been added to the `Cell` class to easier query and modify the `cssClass` property: `addCssClass`, `removeCssClass`, `toggleCssClass`, `hasCssClass`.
+
+== Session API changes
+
+As part of the move toward a stateless server session, the API of the session has been changed.
```

Comment:
Suggestion:

As part of the move toward a stateless backend server, the API of the session has been changed.